app/services/commitments/helper_service.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Commitments
+  class HelperService < BaseService
```

Comment:
I'm not super confident with the naming here. Helper sounds a bit too generic.
I don't have better suggestion, but I think we should avoid this kind of generic services as they does not clearly expose their purpose in their naming